2021/34xxx/CVE-2021-3433.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Expected location (2021/3xxx/CVE-2021-3433.json) of file does not match actual location (2021/34xxx/CVE-2021-3433.json)